app/app.go

```diff
@@ -406,6 +432,11 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		ecocreditMod,
 		dataMod,
 		groupmodule.NewAppModule(appCodec, app.GroupKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
+		// TODO: decide if we want to be a controller chain. For now, we only setup the host.
+		//	this means actions can be executed on Regen Ledger from another chain.
+		// 	however, until we add the controller, Regen Ledger accounts will not be able to execute messages
+		// 	on other chains.
```

Comment:
We should wire up the controlled and make sure it is disabled. This way it could be enabled via governance:

https://github.com/cosmos/ibc-go/blob/main/modules/apps/27-interchain-accounts/controller/types/params.go#L9-L12

Reply:
Currently the command to query the controller is available but it produces the following error:

```
Error: rpc error: code = Unknown desc = value method github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/keeper.Keeper.Params called using nil *Keeper pointer: panic
```

Reply:
yeah the controller is a lot more tricky than the host. it needs middleware, and in the demo i linked in discord, it appears the thing chain devs should be doing is creating their own `x/intertx` module (the one in the demo says its experimental and not to use it in prod) ? but it also looks like it can be wired up using a new `IBCFee` module. 

the docs are also a bit confusing for chain devs, as the [integration guide](https://github.com/cosmos/ibc-go/blob/main/docs/apps/interchain-accounts/integration.md) mentions an ICAAuth module that can be used for the controller's middleware, but that module doesn't appear to exist ??

we may just want to start w/ host for now and wait for them to update the docs to see what chaindevs should do for integrating controllers 🤷🏻

Reply:
Can we make this an issue rather than a todo? No need for the todo here but an issue to track would be great.

Reply:
We can throw it in the backlog. Probably not something we need for v5.0.

Reply:
issue: https://github.com/regen-network/regen-ledger/issues/1453

commented in code: https://github.com/regen-network/regen-ledger/pull/1441/commits/bf6f818dbd8b87c8d114cca2d88d171acec213d7